lib/Transfers/TransfersSynchronizer.cpp

```diff
@@ -120,6 +120,13 @@ void TransfersSyncronizer::addPublicKeysSeen(
     }
 }
 
+void TransfersSyncronizer::markTransactionSafe(const Hash &transactionHash)
```

Comment:
The misspellings was already inside the repo, so just as a note:

Syncronizer / Synchronizer not a big deal